src/main/java/it/gov/pagopa/pu/classification/util/Utilities.java

```diff
@@ -36,4 +43,11 @@ public static boolean isValidIntervalBetweenLocalDate(LocalDate dateFrom, LocalD
   public static String getTraceId(){
     return MDC.get("traceId");
   }
+
+  public static String amountToString(BigDecimal amount) {
+    DecimalFormatSymbols symbols = new DecimalFormatSymbols();
+    symbols.setDecimalSeparator('.');
+    DecimalFormat df = new DecimalFormat("#0.00", symbols);
```

Comment:
define these as constants instead of create them each time

```
private static final decimalFormatSymbols;
static{
  decimalFormatSymbols = ew DecimalFormatSymbols();
  symbols.setDecimalSeparator('.');
}
public static final ThreadLocal<NumberFormat> DECIMAL_FORMAT = ThreadLocal.withInitial(() -> new DecimalFormat("#.00", symbols));
```